### PR TITLE
PPF-75 validate current approach forms inertia

### PR DESCRIPTION
**Pull request by simon-debruijn**
### Added

- react hooks eslint rule

### Changed

- rewrote form with reusable components
- prettier formatting settings

### Fixed

- unresponsive input fields

---
Ticket: https://jira.uitdatabank.be/browse/PPF-75

Screenshot:
Screenshot 2022-12-01 at 10 50 38

### CODE REVIEW

**Review by brampauwelyn — APPROVED** [review]

**Comment by bertramakers** [comment]
@LucWollants I think we should also make @brampauwelyn and @simon-debruijn code owners of this repo so they can approve/merge frontend PRs without explicit approval from you/Jonas/me

**Review by bertramakers — APPROVED** [review]
I haven't done a review as this is not my expertise, but I want to make it able to merge the PR because @brampauwelyn approved

**Review by LucWollants — APPROVED** [review]

**Pull request merged**
